nixos/modules/minimal.nix

```diff
@@ -0,0 +1,205 @@
+/*
+  This is a minimal list of nixos modules.
```

Comment:
"a minimal" is still contradictory, unless we have multiple minimal sets of the same size. I don't believe that's the case.

I'd go with something less absolute and more vague like "core" or "sea level" or something, for the name.